project-manager.rkt

```diff
@@ -66,7 +66,8 @@
          [callback
           (λ (btn event)
             (define path (get-directory #f this))
-            (when (and path (directory-exists? path))
+            (define (get-projects) (file->lines projects-file))
+            (when (and path (directory-exists? path) (not (member (path->string path) (get-projects))))
```

Comment:
Code style: Since there have three conditions right now, prefer
(and A
         B
         C)